sys-apps/busybox/busybox-1.34.1-r1.ebuild

```diff
@@ -348,7 +348,8 @@ pkg_postinst() {
 		cd "${T}" || die
 		mkdir _install
 		tar xf busybox-links.tar -C _install || die
-		echo n | cp -ivpPR _install/* "${ROOT}"/ || die "copying links for ${x} failed"
+		# 907432: cp -n returns error if it skips any file, but that is expected here
```

Comment:
See the mess in https://debbugs.gnu.org/cgi/bugreport.cgi?bug=62572.

But we can't really use any of that for now so I think this is probably fine.

Reply:
Maybe we should put a note to revisit once coreutils XXX is out.